### PR TITLE
Сhanges are related to the club's annual reports, a new logic of display and storage

### PR DESCRIPTION
**Pull request by sharix4**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/pages/AnnualReport/AnnualReportTable/ClubAnnualReportInformation/ClubAnnualReportInformation.tsx [hunk 1]
```diff
@@ -52,18 +52,19 @@ const ClubAnnualReportInformation = () => {
         setIsLoading(true);
         try {
             let clubReport = await getClubAnnualReportById(id);
+            console.log(clubReport);
```
Comment (christykobyletska) [inline src/pages/AnnualReport/AnnualReportTable/ClubAnnualReportInformation/ClubAnnualReportInformation.tsx:55]: please delete console logs here and below

File: src/pages/AnnualReport/AnnualReportTable/ClubAnnualReportInformation/ClubAnnualReportInformation.tsx [hunk 2]
```diff
@@ -52,18 +52,19 @@ const ClubAnnualReportInformation = () => {
...
+
             setClubAnnualReport(clubReport.data.annualreport);
             setStatus(clubReport.data.annualreport.status);
 
-            let response = await getClubMembersInfo(clubReport.data.annualreport.clubId);
-
-            setClub(response.data);
+          //  let response = await getClubMembersInfo(clubReport.data.annualreport.clubId);
```
Comment (christykobyletska) [inline src/pages/AnnualReport/AnnualReportTable/ClubAnnualReportInformation/ClubAnnualReportInformation.tsx:60]: and comments

File: src/pages/AnnualReport/AnnualReportTable/ClubAnnualReportInformation/ClubAnnualReportInformation.tsx [hunk 3]
```diff
@@ -52,18 +52,19 @@ const ClubAnnualReportInformation = () => {
...
 
-            setAdmins(response.data.administration.filter((a: any) => a != null));
-
-            setClubMembers(response.data.members);
-
-            setFollowers(response.data.followers);
+             //setClub(response.data);
```
Comment (christykobyletska) [inline src/pages/AnnualReport/AnnualReportTable/ClubAnnualReportInformation/ClubAnnualReportInformation.tsx:62]: here

File: src/pages/AnnualReport/ClubAnnualReportCreate/ClubAnnualReportCreate.tsx [hunk 4]
```diff
@@ -70,29 +71,28 @@ export const ClubAnnualReportCreate = () => {
...
-
-
+   
     const handleFinish = async (obj: any) => {
-        seeDeleteModal();
-        if (false) {
+        console.log('Admins');
+        console.log(admins);
+        console.log('members');
+        console.log(members);
```
Comment (christykobyletska) [inline src/pages/AnnualReport/ClubAnnualReportCreate/ClubAnnualReportCreate.tsx:79]: here

File: src/pages/AnnualReport/AnnualReportTable/ClubAnnualReportInformation/ClubAnnualReportInformation.tsx (hunk 1 above)
Comment (sharix4) [reply src/pages/AnnualReport/AnnualReportTable/ClubAnnualReportInformation/ClubAnnualReportInformation.tsx]: done

File: src/pages/AnnualReport/AnnualReportTable/ClubAnnualReportInformation/ClubAnnualReportInformation.tsx (hunk 2 above)
Comment (sharix4) [reply src/pages/AnnualReport/AnnualReportTable/ClubAnnualReportInformation/ClubAnnualReportInformation.tsx]: done

File: src/pages/AnnualReport/AnnualReportTable/ClubAnnualReportInformation/ClubAnnualReportInformation.tsx (hunk 3 above)
Comment (sharix4) [reply src/pages/AnnualReport/AnnualReportTable/ClubAnnualReportInformation/ClubAnnualReportInformation.tsx]: done

File: src/pages/AnnualReport/ClubAnnualReportCreate/ClubAnnualReportCreate.tsx (hunk 4 above)
Comment (sharix4) [reply src/pages/AnnualReport/ClubAnnualReportCreate/ClubAnnualReportCreate.tsx]: done

**Review by christykobyletska — APPROVED** [review]

**Review by nicoolle — APPROVED** [review]

**Review by OleksandrBilan — APPROVED** [review]

**Pull request merged**